src/modules/job-manager/event.c

```diff
@@ -158,12 +158,7 @@ void event_publish (struct event *event, const char *topic,
     struct job_manager *ctx = event->ctx;
     flux_future_t *f;
 
-    /* O? support in jansson 2.8 */
-    if (o)
-        f = flux_event_publish_pack (ctx->h, topic, 0, "{s:O}", key, o);
-    else
-        f = flux_event_publish_pack (ctx->h, topic, 0, "{s:n}", key);
-    if (!f) {
+    if (!(f = flux_event_publish_pack (ctx->h, topic, 0, "{s:O?}", key, o))) {
         flux_log_error (ctx->h, "%s: flux_event_publish_pack", __FUNCTION__);
         goto error;
     }
```

Comment:
There's another one of these in `event_batch_pub_annotations()` (searsch for jansson 2.8).

Reply:
Ahh thanks.  I had already removed `event_batch_pub_annotations()` from my other PR :-)  I'll fix it up since its not yet gone in this branch.